multimodal/tests/README.md

```diff
@@ -1,55 +1,66 @@
 # Welcome to Contributing to AutoGLuon-AutoMM!
- To guarantee code quality and correctness, we do two kinds of testing: 
-  - code style check
-  - unit testing
+
+To guarantee code quality and correctness, we do two kinds of testing:
+
+- code style check
+- unit testing
 
 ## Code Style Check
 
 ### black
-[black](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html) is a PEP 8 compliant opinionated formatter with its own style. We use it to organize our code style. This is important to guarantee code quality as we have many developers work on the same codebase. The continuous integration (CI) would fail if your code doesn't meet `black`'s style.
-
 
+[black](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html) is a PEP 8 compliant opinionated formatter with its own style. We use it to organize our code style. This is important to guarantee code quality as we have many developers work on the same codebase. The continuous integration (CI) would fail if your code doesn't meet `black`'s style.
 
 Before submitting a pull request, you can run `black` locally to format your code. First, install it:
 
 ```
 pip install "black>=22.3"
 ```
+
 Then run it:
 
 ```
 black source_file_or_directory --line-length 119
 ```
-We use line length 119 instead of the default. You can refer to the CI's [black configurations](https://github.com/awslabs/autogluon/blob/master/pyproject.toml).
 
-Note that if using `black` as a plugin in your IDE, the plugin may not use the configuration file `pyproject.toml` in our project. So, you need to configure the IDE plugin separately by following [here](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file). 
+We use line length 119 instead of the default. You can refer to the CI's [black configurations](https://github.com/awslabs/autogluon/blob/master/pyproject.toml).
```

Comment:
Would you know why we will have this diff?

Reply:
This diff is a bit strange. There is actually no change here. You can compare the two branches. There are exactly the same. I don't know why GitHub shows that there is diff. 

**Before:**
https://github.com/awslabs/autogluon/blob/master/multimodal/tests/README.md
![image](https://user-images.githubusercontent.com/5016453/188877666-7944407b-54b8-4bd2-b888-f6529b09d608.png)

**After:**
https://github.com/giswqs/autogluon/blob/typos/multimodal/tests/README.md
![image](https://user-images.githubusercontent.com/5016453/188878047-f83e5d62-b811-4af9-8088-e6e4a890ab4e.png)